scripts/hypothesistooling.py

```diff
@@ -0,0 +1,79 @@
+import subprocess
+import os
+
+
+def current_branch():
+    return subprocess.check_output([
+        "git", "rev-parse", "--abbrev-ref", "HEAD"
```

Comment:
Is this for use on a local/dev environment, or is it a more general component?

If more general (incl. Travis), is `git(1)` guaranteed to be present?